### ISSUE TITLE
New icons

### ISSUE THREAD

**Issue by HerbFargus** [botolo78/RetroPie-Manager#6]
The talented Rookervik provided some of his talent: what format are the files needed in in order to implement them? (there is an image here its just a white background)

![logo](https://cloud.githubusercontent.com/assets/10035308/16600045/881eaeb8-42c2-11e6-9322-c730f12f0dde.png)

[RecalBox-Manager(RetroPie).zip](https://github.com/botolo78/RetroPie-Manager/files/348645/RecalBox-Manager.RetroPie.zip)

**Comment by botolo78** [comment]
The icons are stored and used as fonts. You can find the current ones in the icomoon set RetroPie-Manager\project\webapp_statics\fonts and you can use the set to create a new one using the web app 

https://icomoon.io/app/#/select

You can create a complete new one picking from the the icomoon set and adding the new one.
Or you can load the original recalbox icon/svg file (mine has 2 wrong retropie icons we should get rid of) and add the new one.

Keep in mind the new icon must be a SVG file.

When  you are done export the set and it will be zipped with an html/css to show you the css font codes to use with each image.

For example if you look at the extended app.css in RetroPie-Manager\project\webapp_statics\css from row 8106 you should see the font family definition and then from 8276 each icon definition.

**Comment by botolo78** [comment]
btw if you modify the app.css remember to minify the file

**Comment by botolo78** [comment]
Since you are working on the module if you don't have anything against I'll take care of the icons.
I'd like to create multiple css with different color scheme and let the user choose his "theme".

If you like the idea I'll start next week to work on it

**Comment by HerbFargus** [comment]
Yeah that's fine. I'll see if I can get an svg of the above icons for you play with
